source/operations/concepts.rst

```diff
@@ -8,126 +8,173 @@ Core Operational Concepts
    :local:
    :depth: 2
 
-The following core concepts are fundamental to operation of MinIO deployments, including but not limited to installation and management of MinIO.
-
-What Is Object Storage?
------------------------
-
-.. _objects:
-
-An :ref:`object <objects>` is binary data, sometimes referred to as a Binary
-Large OBject (BLOB). Blobs can be images, audio files, spreadsheets, or even
-binary executable code. Object Storage platforms like MinIO provide dedicated
-tools and capabilities for storing, retrieving, and searching for blobs. 
-
-.. _buckets:
-
-MinIO Object Storage uses :ref:`buckets <buckets>` to organize objects. 
-A bucket is similar to a folder or directory in a filesystem, where each
-bucket can hold an arbitrary number of objects. MinIO buckets provide the 
-same functionality as AWS S3 buckets. 
-
-For example, consider an application that hosts a web blog. The application
-needs to store a variety of blobs, including rich multimedia like videos and
-images. The structure of objects on the MinIO server might look similar to the
-following:
-
-.. code-block:: text
-
-   / #root
-   /images/
-      2020-01-02-MinIO-Diagram.png
-      2020-01-03-MinIO-Advanced-Deployment.png
-      MinIO-Logo.png
-   /videos/
-      2020-01-04-MinIO-Interview.mp4
-   /articles/
-      /john.doe/
-         2020-01-02-MinIO-Object-Storage.md
-         2020-01-02-MinIO-Object-Storage-comments.json
-      /jane.doe/
-         2020-01-03-MinIO-Advanced-Deployment.png
-         2020-01-02-MinIO-Advanced-Deployment-comments.json
-         2020-01-04-MinIO-Interview.md
-
-MinIO supports multiple levels of nested directories and objects to support 
-even the most dynamic object storage workloads.
-
-Deployment Architecture
------------------------
-
-:ref:`Erasure Set <minio-ec-erasure-set>`
-   A set of disks that supports MinIO :ref:`Erasure Coding
-   <minio-erasure-coding>`. Erasure Coding provides high availability,
-   reliability, and redundancy of data stored on a MinIO deployment.
-
-   MinIO divides objects into chunks and evenly distributes them among each
-   drive in the Erasure Set. MinIO can continue seamlessly serving read and
-   write requests despite the loss of any single drive. At the highest
-   redundancy levels, MinIO can serve read requests with minimal performance
-   impact despite the loss of up to half (``N/2``) of the total drives in the
-   deployment.
+What are the components of a MinIO Deployment?
+----------------------------------------------
+
+A MinIO deployment consists of a set of storage and compute resources running one or more :mc:`minio server` nodes that together act as a single object storage repository. 
+
+A standalone cluster consists of a single Server Pool with a single :mc:`minio server` node. 
```

Comment:
Can a SNSD deployment be a cluster? I'm a little lost on how we want to use this term.

Reply:
Good catch. Our style guide calls for us to avoid using `cluster` for SNSD, only for distributed deployments. Swapped for instances in this paragraph.